api/v1/models/cluster_status.go

```diff
@@ -16,7 +16,6 @@ import (
 
 // ClusterStatus Status of cluster
 // swagger:model ClusterStatus
-// +k8s:deepcopy-gen=true
```

Comment:
I ran `make generate-api` locally and investigate this.

Reply:
Fixed - this is going to be obnoxious for future API work, though. We will have to manually undo changes to code which we need `deepcopy-gen` functionality for :(

Reply:
Yep, I for one find this pretty confusing. Not sure whether someone has looked into any changes we might make to the tooling to prevent this from happening.

Reply:
I tried to sort it out (I promised aanm I would). Unfortunately, we can't make a separate file that lists which types need a k8s deepcopy generated (one of the other autogenenarated thing can take an annotation on the type, or a list somewhere else). The only thing I can think of is to have a separate script/tool that takes a list of types and inserts the `// +k8s:deepcopy-gen=true` line.